File: sol/1598.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        dep = 0
        for step in logs:
            match step:
                case '../':
                    dep = max(0, dep - 1)
                case './':
                    pass
                case _:
                    dep += 1
        return dep

File: sol/1598/1598.cpp
struct Solution {
    inline int minOperations(vector<string>& logs) {
        int dep = 0;
        for (auto &step : logs){
            if (step == "../"){
                dep = max(0, dep - 1);
            } else if (step != "./"){
                ++dep;
            }
        }
        return dep;
    }
};
